hicexplorer/hicBuildMatrix.py

```diff
@@ -14,12 +14,6 @@
 
 debug = 1
 
-# check pysam version
-from distutils.version import LooseVersion
-if LooseVersion(pysam.__version__) < LooseVersion("0.8.3"):
-    exit("\n*ERROR*\n\nVersion of pysam has to be higher than 0.8.3. Current installed version is {}\n".format(pysam.__version__))
-
-
```

Comment:
Why did you remove this lines? Even though the installer checks that pysam should be higher than 0.8.3, the user could have other version because of PYTHONPATH? I remember I put those lines after some user had a problem.